src/System.Collections.Specialized/src/System/Collections/Specialized/NameObjectCollectionBase.cs

```diff
@@ -591,7 +591,7 @@ void ICollection.CopyTo(Array array, int index)
 
                 if (index < 0)
                 {
-                    throw new ArgumentOutOfRangeException("index", SR.Format(SR.IndexOutOfRange, index.ToString(CultureInfo.CurrentCulture)));
+                    throw new ArgumentOutOfRangeException("index", SR.ArgumentOutOfRange_NeedNonNegNum);
```

Comment:
It seems the original intention in both of these cases was to include the value in the exception.  So shouldn't these instead be:
```
throw new ArgumentOutOfRangeException("index", index, SR.ArgumentOutOfRange_NeedNonNegNum);
```
?